Rebuild-Black-Business/RBB-Website
# Feature/justins logo and 2 Bug Fixes for Businesses Card Styling

# Describe your PR

- Added Justin's logo to About section under his name and linked to https://www.ThugDebugger.dev'
- Fixes business card description/location having wrong style
- Fixes business card description not showing elipses (this happened because substring returns a new string so checking if it's greater than the max did nothing because they were equal! So switched to checking if they matched and showing them

Related to #
Fixes #

## Pages/Interfaces that will change

### Screenshots / video of changes

Drop some screenshots of the before and after of your work here. Better yet, take a screen recording using a tool like [Loom](https://www.loom.com/)

![image](https://user-images.githubusercontent.com/8475305/84458299-b4ce9080-ac32-11ea-99f4-9a477fcf0dca.png)

Businesses styling bug fixes:
![image](https://user-images.githubusercontent.com/8475305/84459589-cfeecf80-ac35-11ea-835f-278bbf2ce751.png)

## Steps to test

1. Go to /about
2. inspect logo under Justin's name

1. Go to /businesses
2. Find a business with description
3. Check the styling
4. Find a business with a long description (> 250 char)
4. Check that it has elipses

### Additional notes
